library/src/main/java/com/novoda/downloadmanager/LiteDownloadBatchStatus.java

```diff
@@ -81,22 +89,29 @@ public long downloadedDateTimeInMillis() {
         return downloadedDateTimeInMillis;
     }
 
+    @WorkerThread
```

Comment:
Why this, if we are calling the async update inside?

Reply:
removed